src/main/java/me/tomsdevsn/hetznercloud/objects/request/LoadBalancerDeleteServiceRequest.java

```diff
@@ -0,0 +1,22 @@
+package me.tomsdevsn.hetznercloud.objects.request;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import me.tomsdevsn.hetznercloud.objects.general.LBService;
+import me.tomsdevsn.hetznercloud.objects.general.LBTarget;
```

Comment:
Cleanup imports

Reply:
I added a minimal checkstyle that checks for unused imports and fixed the errors in various placed. This is PR will now be stale for a few weeks from my side due to summer vacations